.github/workflows/tweet-rfc-update.yml

```diff
@@ -8,9 +8,9 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.action == 'labeled' && contains(github.event.label.name, 'Commenting')
     steps:
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
```

Comment:
Not sure if we use Dependabot yet, but there's a way to have Dependabot automatically update the action versions:

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot